### DISCUSSION TITLE
Want to Incorporate Bengali (Country: Bangladesh) Language

### DISCUSSION THREAD

**Discussion by ShojibFlamon** [andrewdwallo/transmatic#14] [Feature Requests] [answered]
Hello,

I am highly interested in contributing to the inclusion of the Bengali language, which is the official language of Bangladesh. Please consider assigning me to support this new language.

Thank you.

**Comment by andrewdwallo** [comment, marked as answer]
AWS Translate, which is the default engine for this package, already supports the Bengali language (language code `bn`). You can see [here](https://docs.aws.amazon.com/translate/latest/dg/what-is-languages.html#what-is-languages-supported).

Does this answer your question or do you mean something else?
